src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -90,6 +90,12 @@ impl<'a> Transaction<'a> {
             Transaction::Eip1559(_) => Ok(None),
         }
     }
+    fn case(&self) -> i32 {
```

Comment:
return `Result<pb::EthAddressCase, Error>`, and convert like for example above in `fn coin()` using try_from.